### DISCUSSION TITLE
NDI or Optitrack?

### DISCUSSION THREAD

**Discussion by ndbeuk** [PlusToolkit/PlusLib#1065] [Q&A]
Hi! We will run a new project for which we have already a Telemed EchoBlaster ultrasound. We are considering which camera system to use, the NDI Polaris Vega or the Optrack V120:Trio. Do you have any advice? Are both systems equally developed using 3D slicer & Plus toolkit? Thank you!

**Comment by ungi** [comment]
NDI dominates the medical device market. If you expect to require regulatory approval for your system, then it will be much easier with NDI due to lots of predicate devices. OptiTrack has more device options, it's software is more user friendly, and it costs less. OptiTrack can be a more reasonable choice for lab experiments or teaching. Both are fully supported by PLUS + 3D Slicer.
